release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -85,10 +85,13 @@ For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-k
 ==== VMware vSphere region and zone enablement
 You can deploy an {product-title} cluster to multiple vSphere datacenters or regions that run in a single VMware vCenter. Each datacenter can run multiple clusters or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
 
-You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you configure a multiple region and zone environment.
+[IMPORTANT]
+====
+The VMware vSphere region and zone enablement feature is only compatible with a newly installed cluster, because this feature requires the Container Storage Interface (CSI) storage class. A cluster that was upgraded from a previous release does not include the CSI storage class.
```

Comment:
LGTM, but I made a possible suggestion.
```suggestion
The VMware vSphere region and zone enablement feature is only available with a newly installed cluster because this feature requires the Container Storage Interface (CSI) storage class. A cluster that was upgraded from a previous release does not include the required CSI storage class.
```

Reply:
Thanks, @rvanderp3 . Updated.